main.py

```diff
@@ -417,8 +417,10 @@ def mlperf_inference_run_command(spaces, model, implementation, framework, categ
 
         if docker:
             docker_cmd_suffix = f" \\\n{pre_space} --docker --quiet"
-            docker_cmd_suffix += f" \\\n{pre_space} --test_query_count={test_query_count} {extra_docker_input_string} {extra_input_string}"
-            
+            if "scc24" not in extra_variation_tags:
```

Comment:
I think we should have an input argument `skip_test_query_count` so that we don't use scc24 specific conditions in main.py (make this generic).